src/lambda/oembed/oembed.js

```diff
@@ -50,7 +50,7 @@ function handler(event, context, callback) {
         author_name: 'Testing Playground',
         author_url: host,
 
-        html: `<iframe src="${url}" width=${maxwidth} height=${maxheight} scrolling="no" frameborder="0" allowtransparency="true" allowfullscreen="true" title="Testing Playground" style="overflow: hidden; display: block;" loading="lazy" name="testing-playground-${Date.now()}"></iframe>`,
+        html: `<iframe src="${url}" width="${maxwidth}" height="${maxheight}" scrolling="no" frameborder="0" allowtransparency="true" allowfullscreen="true" title="Testing Playground" style="overflow: hidden; display: block;" loading="lazy" name="testing-playground-${Date.now()}"></iframe>`,
```

Comment:
Looks like I misunderstood embed.ly's validator. Quotes are fine, they just didn't like the percentage-based width. I thought I was incorrectly quoting numeric values 😖.

Thanks :+1: